apps/datapuller/src/section.ts

```diff
@@ -0,0 +1,52 @@
+import { ISectionItem } from "@repo/common";
+import { ClassSection, ClassesAPI } from "@repo/sis-api/classes";
+
+import setup from "./shared";
+import { Config } from "./shared/config";
+import mapSectionToNewSection from "./shared/sectionParser";
+import { fetchActiveTerms, fetchPaginatedData } from "./shared/utils";
+
+async function updateSections(config: Config) {
+  const log = config.log;
+  const classesAPI = new ClassesAPI();
+
+  log.info("Fetching Active Terms");
+  const activeTerms = await fetchActiveTerms(log, {
```

Comment:
For updating classes and sections, I would recommend fetching both `Current` and `Next` terms, as future terms are likely to change quite often as well.

Reply:
duplicate comment